Bottom: 31de5714fbd2310660b846ab93594cb34bc51960
Top:    ed4ea15f314be9280f035bdd1c26c0e5dc10decc
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 17:43:34 +0000

Refresh of extend-board-export

---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index a2f2ba6..fb495a5 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -13,7 +13,6 @@
 #include "hid/common/face3d.h"
 #include "hid/common/edge3d.h"
 #include "hid/common/object3d.h"
-#include "data.h"
 
 #include "step_writer.h"
